src/Symfony/Component/DependencyInjection/Compiler/PassConfig.php

```diff
@@ -154,7 +154,7 @@ public function getRemovingPasses()
     /**
      * Gets all passes for the Merge pass.
      *
-     * @return array An array of passes
+     * @return CompilerPassInterface An array of passes
```

Comment:
The other parts of the phpdoc should also be changed, as they also talk about arrays